storybook/src/utils/AspectRatio/AspectRatio.stories.tsx

```diff
@@ -3,19 +3,20 @@
  * Copyright Gemeente Amsterdam
  */
 
+import { aspectRatioOptions } from '@amsterdam/design-system-react/src/common/types'
 import { Meta, StoryObj } from '@storybook/react'
 import { AspectRatio } from './AspectRatio'
 
 const meta = {
   title: 'Utilities/CSS/Aspect Ratio',
   component: AspectRatio,
   args: {
-    aspectRatio: 'x-wide',
+    aspectRatio: '16-9',
   },
   argTypes: {
     aspectRatio: {
       control: 'radio',
-      options: ['2x-wide', 'x-wide', 'wide', 'square', 'tall', 'x-tall'],
+      options: aspectRatioOptions,
```

Comment:
The ‘human friendly’ labels still use slashes, like in the tokens table. Can you add a list of ‘9 / 16’ etc. here and to the prop on `Image`?